app/lib/screens/HomeScreens/todo/ToDoTaskEditor.dart

```diff
@@ -0,0 +1,986 @@
+import 'package:avatar_stack/avatar_stack.dart';
+import 'package:avatar_stack/positions.dart';
+import 'package:effektio/common/store/themes/SeperatedThemes.dart';
+import 'package:effektio/controllers/todo_controller.dart';
+import 'package:effektio/screens/HomeScreens/todo/ToDoTaskAssign.dart';
+import 'package:effektio/screens/HomeScreens/todo/screens/SubscriberScreen.dart';
+import 'package:effektio/widgets/AppCommon.dart';
+import 'package:effektio/widgets/ToDoTaskItem.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_icons_null_safety/flutter_icons_null_safety.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:get/get.dart';
+import 'package:get_time_ago/get_time_ago.dart';
+
+class ToDoTaskEditor extends StatefulWidget {
+  const ToDoTaskEditor({Key? key, required this.item, required this.avatars})
+      : super(key: key);
+
+  final ToDoTaskItem item;
+  final List<ImageProvider<Object>> avatars;
+
+  @override
+  State<ToDoTaskEditor> createState() => _ToDoTaskEditorState();
+}
+
+class _ToDoTaskEditorState extends State<ToDoTaskEditor> {
+  final TextEditingController notesController = TextEditingController();
+  final TextEditingController subtitleController = TextEditingController();
+  RxString? lastUpdated;
+  final settings = RestrictedAmountPositions(
+    maxAmountItems: 5,
+    maxCoverage: 0.7,
+    minCoverage: 0.1,
+    align: StackAlign.left,
+  );
+
+  @override
+  void initState() {
+    super.initState();
+
+    lastUpdated = GetTimeAgo.parse(widget.item.lastUpdated!).obs;
+    notesController.text = widget.item.notes ?? 'Add Notes';
+    subtitleController.text = widget.item.title;
+  }
+
+  @override
+  void dispose() {
+    notesController.dispose();
+    subtitleController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
very long ````build()``` method. Please consider splitting it into smaller methods to increase readability.